doc/api/plotting/index.rst

```diff
@@ -41,21 +41,6 @@ of at least 9.0 and in a future release will require `vtk>=9.2`.
    charts/index
 
 
-Widget API
-----------
-The :class:`pyvista.Plotter` class inherits all of the widget methods in
-:class:`pyvista.WidgetHelper`, so all of the following methods
-are available from any PyVista plotter.
-
-.. currentmodule:: pyvista
-
-.. autosummary::
-   :toctree: _autosummary
-   :template: custom-class-template.rst
-
-   WidgetHelper
```

Comment:
Firstly, I think we could still leave a note here mentioning widgets, pointing users to the Widgets section of the examples.

Secondly, there are a bunch of examples that reference `WidgetHelper.*` methods:
```bash
$ git grep WidgetHelper examples/
examples/03-widgets/box-widget.py::func:`pyvista.WidgetHelper.add_box_widget` and
examples/03-widgets/box-widget.py::func:`pyvista.WidgetHelper.clear_box_widgets` methods respectively.
examples/03-widgets/box-widget.py::func:`pyvista.WidgetHelper.add_mesh_clip_box` method.
examples/03-widgets/checkbox-widget.py:See :func:`pyvista.WidgetHelper.add_checkbox_button_widget` for more details.
examples/03-widgets/line-widget.py::func:`pyvista.WidgetHelper.add_line_widget` and
examples/03-widgets/line-widget.py::func:`pyvista.WidgetHelper.clear_line_widgets` methods respectively.
examples/03-widgets/plane-widget.py::func:`pyvista.WidgetHelper.add_plane_widget` and
examples/03-widgets/plane-widget.py::func:`pyvista.WidgetHelper.clear_plane_widgets` methods respectively.
examples/03-widgets/slider-bar-widget.py::func:`pyvista.WidgetHelper.add_slider_widget` and
examples/03-widgets/slider-bar-widget.py::func:`pyvista.WidgetHelper.clear_slider_widgets` methods respectively.
examples/03-widgets/slider-bar-widget.py:# :func:`pyvista.WidgetHelper.add_mesh_threshold` method which leverages the
examples/03-widgets/sphere-widget.py::func:`pyvista.WidgetHelper.add_sphere_widget` and
examples/03-widgets/sphere-widget.py::func:`pyvista.WidgetHelper.clear_sphere_widgets` methods respectively.
examples/03-widgets/spline-widget.py::func:`pyvista.WidgetHelper.add_spline_widget` and
examples/03-widgets/spline-widget.py::func:`pyvista.WidgetHelper.clear_spline_widgets` methods respectively.
examples/03-widgets/spline-widget.py::func:`pyvista.WidgetHelper.add_mesh_slice_spline`.
```
These should point to the corresponding `Plotter` methods.

Thirdly, there's something weird with the build docs. When I click the main "plotting" index in the sidebar, this is what the sidebar looks like:
![Screenshot from 2022-05-07 12-09-18](https://user-images.githubusercontent.com/17914410/167249776-3d8b9599-6377-4653-a743-c31438ccd1db.png)
"Charts" followed by "Convenience functions" as it should be.

But if I then click on the "Plotting" or "Pyvista and PyQt" submenu, I see this:
![Screenshot from 2022-05-07 12-10-32](https://user-images.githubusercontent.com/17914410/167249816-aca9256f-4cdd-48a6-ba67-83f3dcb2b0d0.png)
Note that `pyvista.WidgetHelper` reappeared.

I can't see what's causing this and it smells like a bug. `git grep WidgetHelper` doesn't show anything weird. Any chance that some cache (partial `clean`) is somehow interfering with an earlier build?

Reply:
First and second points fixed in adf4175752f3c126c1621efa64992723a489a138.

Third point can't be reproduced on my end after running a full clean build.

Reply:
Sorry, I forgot to specify that I see that weird behaviour using the CI build artifact.

Reply:
> Sorry, I forgot to specify that I see that weird behaviour using the CI build artifact.

I don't think it will. The issue is that our cache will only update pages that have changed, and while the links themselves will have changed on the pages that have been updated, the links on the pages that have not been updated won't.

We can reset the cache or just accept that it's fine as-as for the moment.